src/nih_wayfinding/app/scripts/views/navbar/navbar-controller.js

```diff
@@ -54,6 +55,10 @@
             $state.go(stateName, stateParams);
         }
 
+        function onButtonClicked() {
+            $rootScope.$broadcast(NavbarConfig.events.buttonclicked);
```

Comment:
As somewhat of an outsider to this project at the moment, I'm not sure which button this refers to and the naming might be a little generic.